### ISSUE TITLE
Staging Review finding: Button accessible name does not align to text label

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#129517]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** MHV Horizon Team
**Product:** MHV on Va.gov portal
**Feature:** Interstitials for email notification confirmation

## Findings details
**VA.gov Experience Standard - issue:** [User encounters content that does not meet best practices.]()
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)

**Launch-blocking:** No 
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

The accessible name of a control must match or start with the text value of the control, so that speech input users can effectively interact with the element. The "Update email in profile button" has an `aria-label` of "Update contact email", creating a mismatch making voice activation difficult. 

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="1166" height="386" alt="aria label mismatch" src="https://github.com/user-attachments/assets/bd7764fb-8498-47fd-80f1-973a0ee942c0" />

</details>

## Recommended action

It doesn't appear that the `aria-label` is providing additional context, so the recommendation is to remove it. 

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** [WCAG 2.5.3 Label in Name - Level A](https://www.w3.org/TR/WCAG22/#label-in-name)
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Context:** N/A
- **VADS Guidance:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/o3jkwoe) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe this finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
